dotcom-rendering/src/components/Card/components/CardLayout.tsx

```diff
@@ -68,8 +68,9 @@ const decidePosition = (
 	imagePositionOnDesktop: ImagePositionType,
 	imagePositionOnMobile: ImagePositionType,
 	imageType?: CardImageType,
+	containerType?: DCRContainerType,
```

Comment:
It's collapsed in the diff, but if `imagePostionOnDesktop` is not `left` or `right` it falls through to the default condition which sets `flex-direction: row-reverse` on mobile. I'm not especially keen on having to pass in `containerType` either so will have another think about this…

Reply:
Ah, that's a good point! `imagePositionOnMobile` is ignored when there is an avatar, but is set to `left` by default. I've refactored so that if it's set to `top` or `bottom` it forces a vertical layout like it already does on tablet. That's much neater and doesn't require the card layout to know about the specific container type.